### PR TITLE
fix: use new ARM crossbuild docker images

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

It chooses the ARM Docker image to cross-compile based on the architecture.

## Why is it important?

The ARM packages are currently broken.

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have made corresponding change to the default configuration files
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## How to test this PR locally

```
git clone git@github.com:kuisathaverat/beats.git
cd beats/auditbeat
git checkout origin/fix_armv7 -b fix_armv7
mage package
```

## Related issues

closes https://github.com/elastic/beats/issues/25956

### CODE REVIEW

**Review by mdelapenya — APPROVED** [review]

**Review by mtojek — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -352,9 +352,9 @@ def packagingLinux(Map args = [:]) {
                 'linux/amd64',
                 'linux/386',
                 'linux/arm64',
+                'linux/armv7',
```
Comment (urso) [inline Jenkinsfile:355]: Let's keep armv7 disabled by default. We don't publish those officially I think, so we can safe some CPU cycles on CI here.

**Review by kvch — APPROVED** [review]

**Review by urso — APPROVED** [review]

**Comment by kuisathaverat** [comment]
I guess I have to run `make update`

```
[2021-05-31T08:29:28.656Z] mage/crossbuild.go
[2021-05-31T08:29:55.275Z] >> check: Checking source code for common problems
[2021-05-31T08:30:21.897Z] Error: some files are not up-to-date. Run 'make update' then review and commit the changes. Modified: [dev-tools/mage/crossbuild.go]
[2021-05-31T08:30:21.897Z] ../libbeat/scripts/Makefile:152: recipe for target 'check' failed
[2021-05-31T08:30:21.897Z] make: *** [check] Error 1
[2021-05-31T08:30:21.897Z] make: Leaving directory '/var/lib/jenkins/workspace/PR-25987-7-fd802e94-5d5d-414e-949d-d4ee7a50ea36/src/github.com/elastic/beats/dev-tools'
script returned exit code 2
```

**Comment by kuisathaverat** [comment]
I have run `make update` and there are no changes to push so I dunno why the lint of dev-tools fail

**Review by cachedout — APPROVED** [review]

**Comment by kuisathaverat** [comment]
after executing a `make check` the liting stage pass, but there are 3 test on libbeat that fail, they seem not related to the changes in this PR 


#### Test stats :test_tube:

| Test         | Results                         |
| ------------ | :-----------------------------: |
| Failed       | 3  |
| Passed       | 18225  |
| Skipped      | 1479 |
| Total        | 19707   |

</p>
</details>



  ### Test errors [](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-25987/detail/PR-25987/10//tests)
<details><summary>Expand to view the tests failures</summary><p>
    
    
##### `Build&Test / x-pack/libbeat-build / test_configs_cache – x-pack.libbeat.tests.system.test_management.TestManagement`


<ul>
<details><summary>Expand to view the error details</summary><p>

```
 AssertionError: assert 1 == 0 
 ```
</p></details>
<details><summary>Expand to view the stacktrace</summary><p>

```
 self = <test_management.TestManagement testMethod=test_configs_cache>

    @unittest.skipIf(not INTEGRATION_TESTS,
                     "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
    def test_configs_cache(self):
        """
        Config cache is used if Kibana is not available
        """
        # Enroll the beat
        config_path = os.path.join(self.working_dir, "mockbeat.yml")
        self.render_config_template("mockbeat", config_path, keystore_path=self.keystore_path)
        exit_code = self.enroll(self.es_user, self.es_pass)
>       assert exit_code == 0
E       AssertionError: assert 1 == 0

tests/system/test_management.py:172: AssertionError 
 ```
</p></details>
</ul>
    
##### `Build&Test / x-pack/libbeat-build / test_enroll – x-pack.libbeat.tests.system.test_management.TestManagement`


<ul>
<details><summary>Expand to view the error details</summary><p>

```
 AssertionError: assert 1 == 0 
 ```
</p></details>
<details><summary>Expand to view the stacktrace</summary><p>

```
 self = <test_management.TestManagement testMethod=test_enroll>

    @unittest.skipIf(not INTEGRATION_TESTS,
                     "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
    def test_enroll(self):
        """
        Enroll the beat in Kibana Central Management
        """
    
        assert len(glob.glob(os.path.join(self.working_dir, "mockbeat.yml.*.bak"))) == 0
    
        # We don't care about this as it will be replaced by enrollment
        # process:
        config_path = os.path.join(self.working_dir, "mockbeat.yml")
        self.render_config_template("mockbeat", config_path, keystore_path=self.keystore_path)
    
        config_content = open(config_path, 'r').read()
    
        exit_code = self.enroll(self.es_user, self.es_pass)
    
>       assert exit_code == 0
E       AssertionError: assert 1 == 0

tests/system/test_management.py:56: AssertionError 
 ```
</p></details>
</ul>
    
##### `Build&Test / x-pack/libbeat-build / test_fetch_configs – x-pack.libbeat.tests.system.test_management.TestManagement`


<ul>
<details><summary>Expand to view the error details</summary><p>

```
 AssertionError: assert 1 == 0 
 ```
</p></details>
<details><summary>Expand to view the stacktrace</summary><p>

```
 self = <test_management.TestManagement testMethod=test_fetch_configs>

    @unittest.skipIf(not INTEGRATION_TESTS,
                     "integration tests are disabled, run with INTEGRATION_TESTS=1 to enable them.")
    def test_fetch_configs(self):
        """
        Config is retrieved from Central Management and updates are applied
        """
        # Enroll the beat
        config_path = os.path.join(self.working_dir, "mockbeat.yml")
        self.render_config_template("mockbeat", config_path, keystore_path=self.keystore_path)
        exit_code = self.enroll(self.es_user, self.es_pass)
>       assert exit_code == 0
E       AssertionError: assert 1 == 0

tests/system/test_management.py:109: AssertionError 
 ```
</p></details>
</ul>
    
</p></details>

**Comment by kuisathaverat** [comment]
### Test errors [](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2FPR-25987/detail/PR-25987/13//tests)
<details><summary>Expand to view the tests failures</summary><p>
    
    
##### `Extended / metricbeat-windows-10-windows-10 / test_processors – metricbeat.tests.system.test_lightmodules.Test`


<ul>
<details><summary>Expand to view the error details</summary><p>

```
 beat.beat.TimeoutError: Timeout waiting for 'cond' to be true. Waited 10 seconds. 
 ```
</p></details>
<details><summary>Expand to view the stacktrace</summary><p>

```
 self = <test_lightmodules.Test testMethod=test_processors>

    def test_processors(self):
        shutil.copytree(
            os.path.join(self.beat_path, "mb/testing/testdata/lightmodules"),
            os.path.join(self.working_dir, "module"),
        )
    
        with http_test_server() as server:
            self.render_config_template(modules=[{
                "name": "test",
                "metricsets": ["json"],
                "namespace": "test",
                # Hard-coding 'localhost' because hostname in server.server_name doesn't always work.
                "hosts": [f"localhost:{server.server_port}"],
            }])
    
            proc = self.start_beat()
    
>           self.wait_until(lambda: self.output_lines() > 0)

tests\system\test_lightmodules.py:32: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <test_lightmodules.Test testMethod=test_processors>
cond = <function Test.test_processors.<locals>.<lambda> at 0x00000265B03BBC10>
max_timeout = 10, poll_interval = 0.1, name = 'cond'

    def wait_until(self, cond, max_timeout=10, poll_interval=0.1, name="cond"):
        """
        Waits until the cond function returns true,
        or until the max_timeout is reached. Calls the cond
        function every poll_interval seconds.
    
        If the max_timeout is reached before cond() returns
        true, an exception is raised.
        """
        start = datetime.now()
        while not cond():
            if datetime.now() - start > timedelta(seconds=max_timeout):
>               raise TimeoutError("Timeout waiting for '{}' to be true. ".format(name) +
                                   "Waited {} seconds.".format(max_timeout))
E               beat.beat.TimeoutError: Timeout waiting for 'cond' to be true. Waited 10 seconds.

..\libbeat\tests\system\beat\beat.py:362: TimeoutError 
 ```
</p></details>
</ul>
    
</p></details>

**Comment by kuisathaverat** [comment]
GrEEEn 😄

**Pull request merged**